shared/tools/RTGen/src/project/RTGen.Python/Generators/PythonGenerator.cs

```diff
@@ -313,6 +313,17 @@ private bool isMethodTemporarilyDisabled(IMethod method)
             return false;
         }
 
+        private bool methodHasStealRef(IMethod method)
```

Comment:
Please keep all method/function names CamelCase. Not so important, but that's the style used in RTGen (and by most C# coders).